domains/certificates/Query.API/API/Dockerfile

```diff
@@ -22,4 +22,6 @@ WORKDIR /app
 COPY --from=publish /app/publish .
 COPY migrations/API.sql /migrations/API.sql
 
+ENV ASPNETCORE_HTTP_PORTS=80
```

Comment:
Why this now?

Reply:
Morten, from Heimdal, mentioned that this is needed, when upgrading to DOTNET 8.

Reply:
It is required when upgrading to dotnet 8